From 578f370c7e7a9f056aefa062b34590b0aa13bce5 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Tue, 15 Aug 2017 17:27:30 +0800
Subject: [PATCH] Fix build on aarch64/musl

Errors

error: redefinition
 of 'struct iovec'
 struct iovec { void *iov_base; size_t iov_len; };
        ^
Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>

Rebase to 0.170
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

---
 backends/aarch64_initreg.c | 2 +-
 backends/arm_initreg.c     | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- a/backends/aarch64_initreg.c
+++ b/backends/aarch64_initreg.c
@@ -33,7 +33,7 @@
 #include "system.h"
 #include <assert.h>
 #if defined(__aarch64__) && defined(__linux__)
-# include <linux/uio.h>
+# include <sys/uio.h>
 # include <sys/user.h>
 # include <sys/ptrace.h>
 # include <asm/ptrace.h>
--- a/backends/arm_initreg.c
+++ b/backends/arm_initreg.c
@@ -38,7 +38,7 @@
 #endif
 
 #ifdef __aarch64__
-# include <linux/uio.h>
+# include <sys/uio.h>
 # include <sys/user.h>
 # include <sys/ptrace.h>
 /* Deal with old glibc defining user_pt_regs instead of user_regs_struct.  */
